src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -93,6 +93,47 @@ export const getReservesGuardData = formData => {
   };
 };
 
+/**
+ * Converts the treatment date range into an array of objects from just an object
+ * @param {object} treatmentDateRange object containing from/to date range
+ * @returns {array} of treatmentDateRange's
+ */
+
+const transformDateRange = treatmentDateRange => [treatmentDateRange];
+
+/**
+ * Cycles through the list of provider facilities and performs transformations on each property as needed
+ * @param {array} providerFacility array of objects being transformed
+ * @returns {object} containing the new Provider Facility structure
+ */
+const transformProviderFacility = providerFacility => {
```

Comment:
When trying to have it be a data type of array, I believe it asked me for a ViewField, but the display didn't warrant a viewField nor the `Add another` button (at least per requirements). That's why the decision was made to have the transformation. Is there a way to have it be stored as an array, but visually not have that `Add another` button?

Otherwise, I'll definitely update the function to use `.map()` and `set()`